index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css"/>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    </head>
+<body>
+  <main class="Main-body">
+  
+    <header class="Portfolio-header">
+      <nav>
+        <a href="#" target="_parent" class="portfolio-logo">My Portfolio</a>
+        <img src="./Images/Union-button.png" class="mobile-icon"/>
+      </nav>
+    </header>
+
+    <section>
+      <div class="Intro">
+        <h1 class="Greeting">I’m Kamran Glad to see you!</h1>
+        <p class="Description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>        
+           <a href="#" class="connecter">let's connect</a>
+
+        <div class="social-icon">
+        <ul>
+        <li><a href="#"><img src="./Images/Twitter.png" alt="Twitter"></a></li>
+        <li><a href="#"><img src="./Images/Linkedin.png" alt="Linked-in"></a></li>
+        <li><a href="#"><img src="./Images/M.png" alt="M"></a></li>
+        <li><a href="#"><img src="./Images/Github.png" alt="github-logo"></a></li>
+        <li><a href="#"><img src="./Images/Victory.png" alt="victory-sign"></a></li>
+      </ul>
```

Comment:
- [ Optional ] Good job for using `ul` tag to group them, and `li` per icon; each icon has a link `a` tag for social icons, I recommend you add your social `href` to let you connect your social networks with the icons.